ccdb5_api/urls.py

```diff
@@ -12,11 +12,25 @@
 Including another URLconf
     1. Add a URL to urlpatterns:  url(r'^blog/', include('blog.urls'))
 """
-from django.conf.urls import include, url
+import django
 from django.contrib import admin
 
 
-urlpatterns = [
-    url(r'^admin/', include(admin.site.urls)),
-    url(r'^', include('complaint_search.urls', namespace="complaint_search")),
-]
+try:
+    from django.urls import include, re_path
+except ImportError:
+    from django.conf.urls import include
+    from django.conf.urls import url as re_path
+
+if django.VERSION >= (2, 0):
+    app_name = "complaint_search"
+    urlpatterns = [
+        re_path(r'^admin/', admin.site.urls),
+        re_path(r'^', include('complaint_search.urls')),
+    ]
+else:
+    urlpatterns = [
+        re_path(r'^admin/', include(admin.site.urls)),
+        re_path(r'^', include('complaint_search.urls',
+                              namespace="complaint_search")),
+    ]
```

Comment:
`app_name` needs to be moved to `complaint_search/urls.py`. Then this could be simplified to:

```suggestion
urlpatterns = [
    re_path(r'^admin/', admin.site.urls),
    re_path(r'^', include('complaint_search.urls')),
]
```

Because that should work in Django 1.11 and 2.2.

Reply:
Done